pwndbg/arguments.py

```diff
@@ -93,9 +87,9 @@ def get(instruction: PwndbgInstruction) -> List[Tuple[pwndbg.lib.functions.Argum
     # Try to extract the data from GDB.
     # Note that this is currently broken, pending acceptance of
     # my patch: https://sourceware.org/ml/gdb-patches/2015-06/msg00268.html
-    if sym and sym[0]:
+    if sym:
         try:
-            n_args_default = len(sym[0].type.fields())
+            n_args_default = len(sym.type.fields())
```

Comment:
todo, maybe need fix, look above. Patch was applied?
```
    # Try to extract the data from GDB.
    # Note that this is currently broken, pending acceptance of
    # my patch: https://sourceware.org/ml/gdb-patches/2015-06/msg00268.html
```

Reply:
I removed comment, patch is already merged in gdb singe 2016:
https://github.com/bminor/binutils-gdb/blob/c2dbc2929e87557f8bc030f6f010d67b19f99f12/gdb/python/py-type.c#L535-L539